15_perms_lab/permslab.txt

```diff
@@ -1,15 +1,30 @@
-1. Using the file structure in the current directory, write the commands that perform the following:
-
-  - change permission of file src/main.c to r--r--r-- using chmod letter notation
-
-  - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
-
-  - change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
-
-  - change permission of "lib" folder and all files and subdirectories inside it to rwxrwxrwx
-
-2. create a new directory named "test" and inside an empty file named "secret". Change permissions of the file "secret" to ---------. Can you delete the file secret? can you delete the folder test? why?
-
-3. Read the output of "help umask". Explain in your own words what umask does, and how would you use it to set new files permissions to ------r--
-
-
+1. Using the file structure in the current directory, write the commands that perform the following:
+
+
+
+  - change permission of file src/main.c to r--r--r-- using chmod letter notation
+
+    chmod a=r src/main.c or chmod u=r,g=r,o=r src/main.c 
+
+  - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
+
+    chmod 564 include/player.h
+
+  - change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
+
+    chmod 600 Doc/*.*   

```

Comment:
Better to use `chmod 600 Doc/*`, as not all files in unix have a period in their name